docs/how-to/managing-content/ingest-guide.md

```diff
@@ -191,3 +191,92 @@ Our ingest form is organized into Paragraphs - organized groups of metadata fiel
+
+1. Go to **Content > Add Content > Repository Item**.
+2. Fill in the metadata form as you normally would.
+3. For `Content type`, select **Publication Issue**.
+4. Make sure to include a value under `Resource Type`. ==This is required in order for your issues to display properly in the View==. 
+5. Make sure to enter a date in `Date Issued/Date Created` in the Origin Information section of the metadata form. This is necessary to ensure your issue appears in the right place in the accordion foldout display. 
+6. Click **Save**, which should bring you to your newly created Publication Issue object.
+
+Next, Page objects need to be added to your Publication Issue. You can either add each page individually, or run a batch upload via the **Children** tab on the Issue object. Note that this "batch" upload is not the same as the Islandora Workbench "batch operations" process that we describe elsewhere on this site.
+
+To batch upload your Page objects (**_Recommended_**):
+
+1. On the Publication Issue node that you just created, click the **Children** tab. 
+2. To the upper right, click **Batch Upload Children**.<br>
+<br>
+    ![<# Button for batch upload of Page objects #>](/arca-docs/assets/batch-upload-pages.png "Screenshot")
+
+2. Under `Content Type`, select **Repository Item** (note that these are Drupal content types, not Islandora content types). 
+3. Under `Model`, select **Page** (*these* are Islandora content types).
+4. Under `Media Type`, select **Image**.
+5. Under `Usage`, select **Original File**.
+6. Uploaded all relevant image files (i.e. all pages) for your Issue. Enter Alt Text for each.
+7. Click **Finish**.
+8. Return to your Publication Issue object to confirm all pages are appearing in the Mirador viewer. 
+
+>With *Batch Upload Children*, you'll notice that there is no interaction with the metadata form. As it's unlikely you would need to append metadata to your Page objects that could not just be added to the parent Publication Issue object, this method is recommended for efficiency.
+
+If you need to add a Page individually:
+
+1. Go to **Content > Add Content > Repository Item**.
+2. Fill in the metadata form as you normally would.
+3. For `Content type`, select **Page**.
+4. The `Collection(s)` field change to now say `Paged Content or Publication Issue`. Enter the name or node ID of the Publication Issue you just created here.
+5. Click **Save**. 
+6. You should be brought automatically to the *Add Image* page. Upload your image file and click **Save** once it has finished uploading. 
+7. Repeat this process for any other pages in the newspaper issue as necessary.
+
+### Publication Issue (PDFs)
+
+The process for ingesting PDFs as Publication Issues requires an extra step. This is because Islandora's default behaviour is to expect individual Page objects to be nested under a Publication Issue. Because a PDF is an already-compiled series of pages, individual Page objects aren't necessary. 
```

Comment:
More concise:

Ingesting PDFs as Publication Issues requires an extra step. By default, Islandora expects Publication Issues to hold individual Page images. Because a PDF is already a collection of pages, individual Page objects aren't necessary.

Note that Publication Issues with PDFs will display in the PDF.js viewer, *not* the Mirador viewer.